### ISSUE TITLE
Json village data -> messages 24114 or 25000?

### ISSUE THREAD

**Issue by tc-maxx** [FICTURE7/CoCSharp#100]
Hi, 

is there a possibility to get json village data from the messages 24114 or 25000?

Thank you
Maxx

**Comment by knightking10** [comment]
yes.Collect the raw data from proxy and read it

**Comment by aididhaiqal** [comment]
Yes

**Comment by tc-maxx** [comment]
Do you've definition files for these messages?

**Comment by FICTURE7** [comment]
Nope, I don't but if you could provide me with a dump of the messages, I could probably find them.

**Comment by tc-maxx** [comment]
That's cool :) I'll send you two examples later by mail.

(done)
